### PR TITLE
docs: Add Windowing to list of samples

### PR DESCRIPTION
**Pull request by MartinZikmund**

### CODE REVIEW

File: doc/samples.md [hunk 1]
```diff
@@ -430,4 +430,10 @@ Demo of the usage of WebRTC in Uno WebAssembly. This sample establishes a direct
 
 Port of the XAML Brewer WinUI3 SkiaSharp Sample application ([blog post](https://xamlbrewer.wordpress.com/2023/09/25/getting-started-with-skiasharp-in-winui-3/) and [source code](https://github.com/XamlBrewer/XamlBrewer.WinUI3.SkiaSharp.Sample)).
 
-[Browse source](https://github.com/unoplatform/Uno.Samples/tree/master/UI/XamlBrewerUnoApp)
\ No newline at end of file
+[Browse source](https://github.com/unoplatform/Uno.Samples/tree/master/UI/XamlBrewerUnoApp)
+
+### Windowing samples
+
+Provides example on the usage of multi-window APIs in Uno Platform apps.
```
Comment (agneszitte) [inline doc/samples.md:437]:
```suggestion
Provides examples of the usage of multi-window APIs in Uno Platform apps.
```

File: doc/samples.md [hunk 2]
```diff
@@ -430,4 +430,10 @@ Demo of the usage of WebRTC in Uno WebAssembly. This sample establishes a direct
 
 Port of the XAML Brewer WinUI3 SkiaSharp Sample application ([blog post](https://xamlbrewer.wordpress.com/2023/09/25/getting-started-with-skiasharp-in-winui-3/) and [source code](https://github.com/XamlBrewer/XamlBrewer.WinUI3.SkiaSharp.Sample)).
 
-[Browse source](https://github.com/unoplatform/Uno.Samples/tree/master/UI/XamlBrewerUnoApp)
\ No newline at end of file
+[Browse source](https://github.com/unoplatform/Uno.Samples/tree/master/UI/XamlBrewerUnoApp)
+
+### Windowing samples
```
Comment (agneszitte) [inline doc/samples.md:435]: @MartinZikmund can you or @morning4coffe-dev help to add a proper structure (ReadMe file, src folder, and doc/assets folder) for the https://github.com/unoplatform/Uno.Samples/tree/master/UI/WindowingSamples, please

Take PongWars sample as an example:
https://github.com/unoplatform/Uno.Samples/tree/master/UI/PongWars

**Comment by MartinZikmund** [comment]
@agneszitte on it!

File: doc/samples.md (hunk 2 above)
Comment (MartinZikmund) [reply doc/samples.md:435]: Done!

**Comment by agneszitte** [comment]
@MartinZikmund fyi I just pushed two additional commits to move the solution files into a src folder and to update to the latest stable versions to have the latest fixes.

For example to have the fix for the pixelated images:
**Before**
image
**After**
image

**Review by agneszitte — APPROVED** [review]

**Comment by agneszitte** [comment]
@MartinZikmund Thank you a lot for the ReadMe and adjustments!
FYI I just pushed additional commits:
- To move the solution files into a src folder
- To keep alphabetical order for the samples list doc
- To update to the latest stable versions to have the latest fixes

For example to have the fix for the pixelated images for Desktop:
**Before**
image
**After**
image

**Comment by agneszitte** [comment]
@MartinZikmund FYI I noticed something when doing some last tests to make sure everything was fine with the latest stable versions. The Window.Title value is not properly Bind for Desktop:
image
If you can take a quick look tomorrow and open an issue for it if needed in Uno please

**Comment by MartinZikmund** [comment]
@agneszitte that is most likely fixed by `x:Bind` adjustments that Youssef did and lifecycle changes on Skia, I will test later and bump Uno version if needed

**Pull request merged**